pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 4.0.0
 homepage: https://github.com/parse-community/Parse-SDK-Flutter
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: '>=2.19.2 <3.0.0'
```

Comment:
`sdk: '>=2.19.2 <3.0.0'` 
I do not recommend making the constraints that tight unless you have a good reason.. like language-specific features that you are using and only work if we are on `>=2.19.2 `

if you do not have any language-specific features or other dependence requires `>=2.19.2`
I suggest making the constraints a little bit looser:
`sdk: '>=2.17.0 <3.0.0'`

Reply:
It was done